### DISCUSSION TITLE
M1 Pro Support at release?

### DISCUSSION THREAD

**Discussion by iXizz3l** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#412] [Q&A]
Hi, I am upgrading from my old Mid 2014 MacBook to the new M1 Pro one. I quickly run through my battery with my old device and now want to use AlDente to help batterie life. Is it save to uninstall when the new one arrives and does it even work or will it be better to wait for an updated version?

**Comment by MatthiasKerbl** [comment]
> Hi, I am upgrading from my old Mid 2014 MacBook to the new M1 Pro one. I quickly run through my battery with my old device and now want to use AlDente to help batterie life. Is it save to uninstall when the new one arrives and does it even work or will it be better to wait for an updated version?

Hi @iXizz3l,

We are trying to get our hands on the new M1 Pro MacBook as quickly as possible but we don’t know when and if AlDente will be compatible with the new devices.

Cheers,
Matthias
